stable/chaoskube/templates/deployment.yaml

```diff
@@ -28,6 +28,15 @@ spec:
         {{- if not .Values.dryRun }}
         - --no-dry-run
         {{- end }}
+        {{- if .Values.excludedWeekdays }}
+        - --excluded-weekdays={{ .Values.excludedWeekdays }}
+        {{- end }}
+        {{- if .Values.excludedTimesOfDay }}
+        - --excluded-times-of-day={{ .Values.excludedTimesOfDay }}
+        {{- end }}
+        {{- if .Values.timezone }}
+        - --timezone={{ .Values.timezone }}
```

Comment:
If you leave out the `if` statements and don't comment out the configs in `values.yaml` (similar to `--namespaces`, `labels` etc.) it will look like the following in your final Pod and should be equivalent:
```yaml
args:
  - --labels=
  - --excluded-weekdays=
```
Makes this file a little easier to read but you may dislike the final `PodSpec` in this way.

I'm fine with with whatever you prefer but I think we should keep the style consistent.

Reply:
From a `PodSpec` perspective, I can see that point, but it doesn't quite matter to me at this time.  I figured (and tested) this way but am happy to maintain existing style -- I'll update it in the next commit to match the style for `--labels` and `--annotations`